notification-manager/src/main/resources/application.yaml

```
@@ -1,3 +1,16 @@
 spring:
   application:
-    name: notification-manager
\ No newline at end of file
+    name: notification-manager
+  mail:
+    host: smtp.gmail.com
+    port: 587
+    username: sukharevroman96@gmail.com
```

don't expose username.
```suggestion
    username: "*****"
```
also add password over here.

When you want to run it on local replace "****"  with your real data but don't commit it